backend/protocol_rpc/requirements.txt

```diff
@@ -17,7 +17,6 @@ SQLAlchemy[asyncio]==2.0.35
 alembic==1.13.2
 eth-account==0.13.3
 eth-utils==5.0.0
-sentence-transformers==3.1.1
```

Comment:
@kp2pml30 please also remove the installation of the torch packages on line 9 on Dockerfile.backend

if you move your work to this repo, I could create branches with minor changes like this for you :)

Reply:
done in 9920b934a0bc0218b1c4cfb9b23303904d7a7507

Reply:
Added permissions to push